composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "shopgate/cloud-integration-shopware",
+    "description": "The cloud API integration for Shopware",
+    "version": "1.0.0-alpha",
+    "require": {
+        "php": "^5.6 || ~7.1"
```

Comment:
I guess 5.6 is to high. PHP 5.4 or 5.3 seems to be more reasonable. Btw. Shopware 4.0.1 requires PHP 5.3.

Reply:
As mentioned in the description, the version to test against is latest. It's not a requirement to support 4.0.1, this is just for proof of concept and library test. If you say I need to support lower versions then maybe half of the code will need to be scratched as it uses modules that don't exist in lower versions ^_^. Same goes for php comments.